docs/user/filenodes.md

```diff
@@ -0,0 +1,56 @@
+# FlowForge File Nodes
+
+Projects running within FlowForge include a set of nodes that make it possible
+to store files safely regardless of the environment. For edge devices the files 
+are stored locally, containerized applications can read and write to persistent 
+storage via API calls to a file-server instance by using these nodes.
+
+*NOTE: This set of nodes allow you to read and write persistent files in a containerised
```

Comment:
User documentation; how do I as a user know this?

Reply:
@hardillb Please consider end-users each time you're implementing config options or multiple backends. It keeps creating unclear situations and it's not great for anyone except the super-power users. We should improve the lives of the 95% of non-power users first after we're able to improve the last 5% too.

Persistent context has now multiple choices, files too, going forward let's have one and only one implementation.